### PR TITLE
Added systemd service file

### PR DESCRIPTION
**Pull request by icodragon**
The system file allows you to restart the node in case of a crash

### CODE REVIEW

File: poolparty/init_poolparty.sh [hunk 1]
```diff
@@ -170,5 +170,28 @@ lsof -i tcp:${PORT_NUMBER} | awk 'NR!=1 {print $2}' | xargs kill
 PORT_NUMBER=26557
 lsof -i tcp:${PORT_NUMBER} | awk 'NR!=1 {print $2}' | xargs kill
 
-
-sh $launch_file
+read -p "Create a systemd service file to run the node? [y/n] " is_create_systemd_file
```
Comment (sampocs) [inline poolparty/init_poolparty.sh]: Can we align this prompt with the styling of the rest of the script?

```
read -p "$(printf $PURPLE"\nCreate a systemd service file to run the node? [y/n] "$NC)" is_create_systemd_file
```

File: poolparty/init_poolparty.sh [hunk 2]
```diff
@@ -170,5 +170,28 @@ lsof -i tcp:${PORT_NUMBER} | awk 'NR!=1 {print $2}' | xargs kill
...
+then
+    tee /etc/systemd/system/strided.service > /dev/null <<EOF
+[Unit]
+Description=Strided Node
+After=network.target
+
+[Service]
+User=$USER
+Type=simple
+ExecStart=$(which strided) start
```
Comment (sampocs) [inline poolparty/init_poolparty.sh]: Let's change this to 
```
ExecStart=$BINARY start
```
so for those who have not installed it under their PATH

File: poolparty/init_poolparty.sh [hunk 3]
```diff
@@ -170,5 +170,28 @@ lsof -i tcp:${PORT_NUMBER} | awk 'NR!=1 {print $2}' | xargs kill
 PORT_NUMBER=26557
 lsof -i tcp:${PORT_NUMBER} | awk 'NR!=1 {print $2}' | xargs kill
 
-
-sh $launch_file
+read -p "Create a systemd service file to run the node? [y/n] " is_create_systemd_file
+if [[ "$is_create_systemd_file" =~ ^([Yy])$ ]]
+then
+    tee /etc/systemd/system/strided.service > /dev/null <<EOF
```
Comment (sampocs) [inline poolparty/init_poolparty.sh:176]: Can we add a dependency check here for `systemd`? You can see the top of the script for a reference for how it was done with the other dependencies

**Review by sampocs — COMMENTED** [review]
Thanks @icodragon, this is a great addition! 

Please see a couple minor comments below

File: poolparty/init_poolparty.sh (hunk 1 above)
Comment (icodragon) [reply poolparty/init_poolparty.sh]: Yes, I will add

File: poolparty/init_poolparty.sh (hunk 2 above)
Comment (icodragon) [reply poolparty/init_poolparty.sh]: I agree, I'll fix it

File: poolparty/init_poolparty.sh (hunk 3 above)
Comment (icodragon) [reply poolparty/init_poolparty.sh:176]: From my experience, as I know `systemd` is not available in MacOS. We can of course add it to dependencies, but this command is **Ubuntu** specific

File: poolparty/init_poolparty.sh (hunk 1 above)
Comment (icodragon) [reply poolparty/init_poolparty.sh]: `lsof -i tcp:${PORT_NUMBER} | awk 'NR!=1 {print $2}' | xargs kill`
This command don't work on Ubuntu. Maybe rewrite them?

Comment (sampocs) [reply poolparty/init_poolparty.sh]: You’re right it doesn’t - but we should have the fix in our latest main. Can you pull in the most recent to your branch and confirm the update worked? (Should be `xargs -r`)

File: poolparty/init_poolparty.sh (hunk 3 above)
Comment (sampocs) [reply poolparty/init_poolparty.sh:176]: My thinking was I’d like it to also work in docker, where it likely isn’t installed by default. That said, if someone is choosing yes for this option, it’s probably safe to assume they have systemd installed, so think its fine to skip the dependency check

File: poolparty/init_poolparty.sh (hunk 1 above)
Comment (icodragon) [reply poolparty/init_poolparty.sh]: Yes, it's all good now. The command works on ubuntu

**Pull request merged**